src/shared/constants.ts

```diff
@@ -92,8 +92,11 @@ export const COPY_TO_CLIPBOARD_INFO_TIMEOUT_MS = 5000
 export const LAMBDA_PACKAGE_TYPE_IMAGE = 'Image'
 
 // URLs for App Runner
-export const APPRUNNER_CONNECTION_HELP_URL =
+export const apprunnerConnectionHelpUrl =
     'https://docs.aws.amazon.com/apprunner/latest/dg/manage-create.html#manage-create.create.github'
-export const APPRUNNER_CONFIGURATION_HELP_URL = 'https://docs.aws.amazon.com/apprunner/latest/dg/manage-configure.html'
-export const APPRUNNER_RUNTIME_HELP_URL = 'https://docs.aws.amazon.com/apprunner/latest/dg/service-source-code.html'
-export const APPRUNNER_PRICING_URL = 'https://aws.amazon.com/apprunner/pricing/'
+export const apprunnerConfigHelpUrl = 'https://docs.aws.amazon.com/apprunner/latest/dg/manage-configure.html'
```

Comment:
Nit: most of the url constants in this file are camelCase.

Probably we should never use uppercase for constants, it doesn't help much. The convention is from C and even there it's only used for macro defines, not `const`.

Reply:
It's pretty arbitrary but the all uppercase is a common convention for global constant _values_ (or at least this is how I interpret it). The `const` keyword is a constant reference within a scope but that does not mean the computed value is the same across executes of a code block within the currently running application. Basically, global + `const` + primitive value -> use UPPERCASE. I sometimes use UPPERCASE for objects too but _technically_ it can still be modified.

I prefer the UPPERCASE for global constant values but consistency wins here.

Reply:
> Basically, global + `const` + primitive value -> use UPPERCASE.

Is this actually helpful though?

Reply:
Not really. I guess it's a hint to the reader of where the value might live, but it's super easy to just have the language server take you there. At this point it's a personal preference.